packages/lib/src/Base.svelte

```diff
@@ -13,19 +13,15 @@
   const translator = initTransaltor(lang || htmlLang || 'en');
 
   let sesamyDesignTokens = `
-    * {
-      --s-main-color: var(--sesamy-main-color, ${hexToHsl('#7D68F4')});
-      --s-bg-color: var(--sesamy-bg-color, ${hexToHsl('#F6DFDC')});
-      --s-bg-opacity: var(--sesamy-bg-opacity, unset);
-      --s-font-family: var(--sesamy-font-family, Helvetica);
-    }
-
```

Comment:
Settings these variables on `*` attaches the styles to every element in the shadow dom. This is simply bad practice. Moved then into the scope of `.base` instead.